### PR TITLE
Admin teleportation of things is now logged.

### PR DESCRIPTION
**Pull request by ShiftyRail**
[administration]

Collect your deadmin valids today.

### CODE REVIEW

**Comment by Kurfursten** [comment]
The real logging change we need is to stop sending adminwarns every time a miner shoots a roid monster. It should simply not send an adminwarn when someone shoots at a roidmonster type on z5. Very narrow scope. Keep shooting at monsters in other contexts.

I should note this really has nothing to do with your PR, it's just my nuke-esque ramble for the day.

File: code/datums/datumvars.dm [hunk 1]
```diff
@@ -779,6 +779,7 @@ function loadPage(list) {
 			M.teleport_to(T)
 		else
 			A.forceMove(T)
+		log_admin("[key_name(usr)] has teleported [A] at ([T.x], [T.y], [T.z]).")
```
Comment (Intigracy) [inline code/datums/datumvars.dm]:
```suggestion
		log_admin("[key_name(usr)] has teleported [A] to [formatLocation(T)].")
```

Comment (Intigracy) [reply code/datums/datumvars.dm]: This proc gives the area name and the coordinates of the destination turf, please use it instead.

**Comment by Intigracy** [comment]
I edited that comment after posting it, if you want to do it that way you'll have to add `var/turf/AT = get_turf(A)` above or below where the other get_turf is on line 776

I'd have suggested the edit in the comment but it doesn't let me do it that far up.

**Comment by SGuivenne** [comment]
> The real logging change we need is to stop sending adminwarns every time a miner shoots a roid monster. It should simply not send an adminwarn when someone shoots at a roidmonster type on z5. Very narrow scope. Keep shooting at monsters in other contexts.
> 
> I should note this really has nothing to do with your PR, it's just my nuke-esque ramble for the day.

But how will you know when the miners try using an artifact gun and are shooting mobs with a practice laser gun before dying screaming?

**Pull request merged**